lib/rucio/daemons/hermes/kafka/hermesk.py

```diff
@@ -0,0 +1,766 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm wondering if you can't express this as a patch on the source version that you have locked.  It would be a lot clearer than copying the entire file.  As it is, I can't easily tell what's your code and what's Rucio's.

Reply:
Understood.  After this PR is completed, I'll change this to a patch.

Reply:
Added a .txt patch file so you can see what was done.  The original hermes.py code was in release 33.0.0.

https://github.com/lsst-dm/ctrl_rucio_ingest/blob/11a8060c77f87edce3a5e6c353b0c203edd51f1a/lib/rucio/daemons/hermes/kafka/hermesk_patch.txt